goth/runner/download/__init__.py

```diff
@@ -286,12 +213,7 @@ def _get_latest_release(
         Only the versions with `tag_name` that contains `self.tag_substring`
         as a substring are considered.
         """
-        url = f"{self.repo_url}/releases"
-        logger.debug("Fetching releases. url=%s", url)
-        response = self.session.get(url)
-        response.raise_for_status()
-
-        all_releases = response.json()
+        all_releases = self.gh_api.repos.list_releases()
         logger.debug("releases=%s", all_releases)
```

Comment:
This is a minor one, but perhaps it's possible to print this in a different format (e.g. JSON or Python repr)? Currently the debug logs look something like this:
```
2021-03-26 19:24:17,613 DEBUG    goth.runner.download                releases=[- url: https://api.github.com/repos/golemfactory/yagna/releases/40429724
- assets_url: https://api.github.com/repos/golemfactory/yagna/releases/40429724/assets
- upload_url: https://uploads.github.com/repos/golemfactory/yagna/releases/40429724/assets{?name,label}
- html_url: https://github.com/golemfactory/yagna/releases/tag/v0.6.3
- id: 40429724
- author:
  - login: github-actions[bot]
  - id: 41898282
  - node_id: MDM6Qm90NDE4OTgyODI=
  - avatar_url: https://avatars.githubusercontent.com/in/15368?v=4
  - gravatar_id:
  - url: https://api.github.com/users/github-actions%5Bbot%5D
  - html_url: https://github.com/apps/github-actions
  - followers_url: https://api.github.com/users/github-actions%5Bbot%5D/followers
  - following_url: https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}
  - gists_url: https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}
  - starred_url: https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}
  - subscriptions_url: https://api.github.com/users/github-actions%5Bbot%5D/subscriptions
  - organizations_url: https://api.github.com/users/github-actions%5Bbot%5D/orgs
  - repos_url: https://api.github.com/users/github-actions%5Bbot%5D/repos
  - events_url: https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}
  - received_events_url: https://api.github.com/users/github-actions%5Bbot%5D/received_events
  - type: Bot
  - site_admin: False
- node_id: MDc6UmVsZWFzZTQwNDI5NzI0
- tag_name: v0.6.3
...
```

You can test this locally by using one of our scripts in the repo, e.g.: `./scripts/download_release.py -v yagna` from the project root.

Reply:
converted to JSON string using https://fastcore.fast.ai/xtras.html#obj2dict